packages/flutter_tools/test/general.shard/dart_plugin_test.dart

```diff
@@ -567,6 +519,22 @@ void main() {
           flutterManifest,
           fs,
           <String, String>{
+          'url_launcher_android': '''
```

Comment:
nit: indentation

Reply:
Fixed (throughout the file; this was copied from an existing test)